.github/workflows/release.yaml

```diff
@@ -27,14 +29,21 @@ jobs:
         uses: imranismail/setup-kustomize@v1
         with:
           kustomize-version: "3.5.4"
-      - name: manifest-debug
-        run: make manifest
-      - name: find-debug
-        run: find ./out
+      - name: Set up Docker Buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          buildx-version: latest
+          qemu-version: latest
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v2.0.2
         with:
           version: latest
           args: release --rm-dist --debug
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Update docker manifest
+        run: |
+          buildx imagetools create \
+            -t packethost/cluster-api-provider-packet:v0.3.0 \
+            packethost/cluster-api-provider-packet:$VERSION-amd64 \
```

Comment:
Is this building the multi-arch manifest? Could we capture this? We now have the version hard-coded in here, and the specific architectures hard-coded in here.

Reply:
ehehe it is my error the `VERSION` env var I set above and you said was not used should be used here :D

Reply:
Can we remove the list of architectures, generate it somehow, so that we don't have it hard-coded here and in Makefile and in multiple yaml, and...

Reply:
As we spoke on Slack, I am not 100% sure about how to do it effectively but I share your same concerns in terms of having configuration duplicated across many tools: makefile, github actions and goreleaser. but I will try to fix it later